controllers/user/GroupController.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace humhub\modules\smartVillage\controllers\user;
+
+
+use humhub\modules\smartVillage\components\AuthBaseController;
+use humhub\modules\user\models\User;
+use \humhub\modules\user\models\Group;
+use Yii;
+
+class GroupController extends AuthBaseController
+{
+    /**
+     *  By Default humhub default group is USER group whose id is 2,
+     */
+    const USER_DEFAULT_GROUP_ID = 2;
+
+    /**
+     * Adds the user to a particular humhub group by passing groupID and userID
+     *
+     * @param $id
+     * @param $userId
+     * @return array
+     */
+    public function actionMemberAddWithoutAuth($id,$userId)
```

Comment:
isn't this another topic and not user registration? or am i wrong?

Reply:
Hi @donni106 
> New users should be added to the HumHub "Default Group" automatically.

As you mentioned your requirement in issue 2. This controller is used to add registered user in humhub default group which is 'USER' Group whose has id=2.
![image](https://user-images.githubusercontent.com/103625795/165942012-c222637a-3f69-4d34-b240-c771a5bf6788.png)